app/views/static/front.haml

```diff
@@ -65,11 +65,10 @@
   .row
     .col-md-4.col-sm-4.col-xs-12
       %h3
-        %i.fa.fa-fw.fa-users.text-primary
-        Groups
+        %i.fa.fa-fw.fa-list.text-primary
+        Lists
       %p
-        Want to ask a question to a specific set of people? No problem!
-        Set up unique groups of users and send them questions apart from everyone else.
+        ??????????
```

Comment:
This should be filled or removed before merging.

Reply:
Maybe @pixeldesu has an idea what to place there now.

Reply:
_"Want to keep an oversight of things? Organize users in Lists to create timelines of their answers."_

Or something along those lines.

Reply:
Sounds good.  I'll take that